setup.py

```diff
@@ -2,7 +2,7 @@
 setup(
     name='unit-python-sdk',
     packages=['unit', 'unit.api', 'unit.models', 'unit.utils'],
-    version='0.7.4',
+    version='0.7.3',
```

Comment:
That's ok, you can remove that. 
We will update the version before the deployment.